models/AssignmentModel.js

```diff
@@ -0,0 +1,64 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+
+const AssignmentSchema = new Schema({
+  assignmentName: {
+    type: String,
+    required: true
+  },
+  AssignmentDescription: {
```

Comment:
I don't believe that this should be capitalized

Reply:
fixed